api/type.rb

```diff
@@ -84,6 +84,12 @@ def to_s
       JSON.pretty_generate(self)
     end
 
+    def lineage
```

Comment:
Can you comment about what this is? I'm pretty sure from code but confirmation would be nice.